src/assets/scss/responsive.scss

```diff
@@ -1,258 +1,247 @@
-@media only screen and (max-width: 600px){
-    .sideDiv {
-        display: none !important;
-    }
-    ._smallScreenText {
-       
-        font-size: 15px !important;
-      
-    }
-    .noPaddingSmallScreen{
-        padding: 0!important;
-    }
-    .colon{
-        display: none;
-    }
-    // .playStoreLogo{
-    //     width: 100% !important;
-    // }
-    .order1{
-        order: 2 !important;
-    }
-    .order2{
-        order: 1 !important;
-    }
-    .routingDiv {
-        // position: relative;
-        // height: calc(100vh - 52px);
-        width: 100% !important;
-        // float: right;
-        // overflow-y: auto;
-    }
-    .wrapper {
-        margin-top: 79px !important;
-    }
-    .navbar{
-        display : none !important;
-    }
-    .responsiveNavbar{
-        display: flex !important;
-    }
-    .alingCenterlineParentHeading {
-        display: flex;
-        align-items: flex-start !important;
-        flex-direction: column;
-        margin-bottom: 10px;
-    }
-    .infoAlingCenterLine{
-        display: inline !important;
-    }
-    .mat-content{
-        display :inline !important;
-    }
-    .bigTabel{
-        display: none !important;
-    }
-    ._responsiveTabel{
-        display:inline !important;
-    }
-    
-    .mat-expansion-panel-header{
-        align-items: baseline !important;
-    }
-    .widthAdj {
-        width: 200px !important;
-        font-size: 10px;
-    }
-    .notResponsiveSurveyButton{
-        display: none !important;
-    }
-    .responsiveSurveyButton{
-        display: flex !important;
-        margin-bottom: 15px;
-    }
-    .responsiveNextButton{
-        position: absolute !important;
-        display: flex!important;
-    }
-    .responsiveQuestion{
-        font-size: 15px;
-    }
-    .checkBoxAdj {
-        margin: 10px 10px!important ;
-    }
-    .radio{
-        font-size: 10px !important;
-    }
-    .checkBox{
-        font-size: 10px !important;
-    }
-    .select{
-        font-size: 10px !important;
-    }
-    .questionFooter {
-        z-index: 0 !important;
-        height: 0!important;
-        position: relative !important;
-    }
-    .card{
-        padding: 0px;
-    }
-    .justifyContentflexStart{
-        justify-content: flex-start !important;
-    }
-    ._halfWidth{
-        width: 100%;
-    }
-    .dashboard{
-        display: flex !important;
-        justify-content: center !important;
-        width:100%;
-    }
+@media only screen and (max-width: 600px) {
```

Comment:
is this responsive changes required ??

Reply:
Yes after bootstrap changes, still some changes not applied correctly